frontend/packages/kubevirt-plugin/src/selectors/vm-like/nextRunChanges.tsx

```diff
@@ -0,0 +1,106 @@
+import * as React from 'react';
+import { VMWrapper } from '../../k8s/wrapper/vm/vm-wrapper';
+import { VMIWrapper } from '../../k8s/wrapper/vm/vmi-wrapper';
+import * as _ from 'lodash';
+import { BootableDeviceType } from '../../types/types';
+import { VMKind, VMIKind } from '../../types/vm';
+import { Alert, AlertVariant } from '@patternfly/react-core';
+import { MODAL_RESTART_IS_REQUIRED } from '../../strings/vm/status';
+import { IsPendingChange } from '../../components/vms/types';
+import { getBootableDevicesInOrder } from '../vm/devices';
+import { getVMIBootableDevicesInOrder } from '../vmi/devices';
+import { mapCDsToSource } from '../../components/modals/cdrom-vm-modal/helpers';
+import { K8sResourceKind } from '../../../../../public/module/k8s/types';
+
+import './nextRunChanges.scss';
+
+export const isFlavorChanged = (vm: VMWrapper, vmi: VMIWrapper): boolean => {
+  if (!vm || !vmi) {
+    return false;
+  }
+
+  return (
+    vm.getFlavor() !== vmi.getFlavor() ||
+    !_.isEqual(vm.getMemory(), vmi.getMemory()) ||
+    !_.isEqual(vm.getCPU(), vmi.getCPU())
+  );
+};
+
+export const isCDROMChanged = (
```

Comment:
these methods should be generic so they can be used by disks as well. Nevertheless the cdrom modal will be deprecated soon.

Reply:
when is 'soon'? 2.5 ? 2.6 ?
if it's deprecated soon should we ignore the changes made in the cdroms at the details tab?
@jelkosz @matthewcarleton please advise

Reply:
not that soon :) 4.6